### ISSUE TITLE
Address issues identified by the July 2020 Sprint

### ISSUE THREAD

**Issue by ghobona** [opengeospatial/ogcapi-maps#47]
The issues are in the annotated PDF at https://github.com/opengeospatial/OGC-API-Sprint-July-2020/blob/master/docs/20200728_Sprint_Review_Of_OAPI_Maps.pdf

**Comment by jerstlouis** [comment]
I went through the PDF and most identified issues had already been resolved.
The two remaining issues that I identified were:

- Common URIs that I think we already discussed fixing but still I went ahead and fixed in #93
- Dependency of Geospatial Data Maps and Dataset Maps on Common - Part 2. But should it actually be on Part 1 for Dataset Maps? We had also touched on this issue at a recent meeting (the dependency is also missing from _Tiles_ [DataSet TileSets](http://docs.ogc.org/DRAFTS/20-057.html#rc_datasetTileSets) and [GeoData TileSets](http://docs.ogc.org/DRAFTS/20-057.html#rc_geoDataResourceTileSets)).

After these couple issues are resolved I think we will be able to close this.
